installed.html

```diff
@@ -1,31 +1,15 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <title>Thanks for Using Font Changer</title>
+    <title>Thanks for using Google Font Changer</title>
```

Comment:
Using word "Google" may cause trouble, because we're not legally entitled to use that font.

Reply:
Addressed and changed in new commits